assembly/pom.xml

```diff
@@ -254,6 +254,14 @@
           <artifactId>spark-hadoop-cloud_${scala.binary.version}</artifactId>
           <version>${project.version}</version>
         </dependency>
+        <!--
+        Redeclare this dependency to force it into the distribution.
+        -->
+        <dependency>
+          <groupId>org.eclipse.jetty</groupId>
```

Comment:
This kinda sucks. Doesn't this also end up pulling up a bunch of other jetty stuff into the packaging?

I guess there's no way around it until Hadoop itself shades jetty in some way...

Reply:
It doesn't pull in anything else. There's already one of the jetty- JARs in the dist/jars directory BTW.

bq. I guess there's no way around it until Hadoop itself shades jetty in some way...

Or when @aajisaka & colleagues implement the Java 9 support and everyone runs to it. This is one of those examples why, from a packaging and deployment perspective, Java 9 is the good one